src/OutputWriters/netcdf_output_writer.jl

```diff
@@ -59,138 +56,144 @@ function write_grid(model; filename="./grid.nc", mode="c",
         "zF" => collect(model.grid.zF)[1:end-1]
     )
 
-    dim_attrib = Dict(
-        "xC" => ["longname" => "Locations of the cell centers in the x-direction.", "units" => "m"],
-        "yC" => ["longname" => "Locations of the cell centers in the y-direction.", "units" => "m"],
-        "zC" => ["longname" => "Locations of the cell centers in the z-direction.", "units" => "m"],
-        "xF" => ["longname" => "Locations of the cell faces in the x-direction.",   "units" => "m"],
-        "yF" => ["longname" => "Locations of the cell faces in the y-direction.",   "units" => "m"],
-        "zF" => ["longname" => "Locations of the cell faces in the z-direction.",   "units" => "m"]
+    dim_attribs = Dict(
+        "xC" => Dict("longname" => "Locations of the cell centers in the x-direction.", "units" => "m"),
+        "yC" => Dict("longname" => "Locations of the cell centers in the y-direction.", "units" => "m"),
+        "zC" => Dict("longname" => "Locations of the cell centers in the z-direction.", "units" => "m"),
+        "xF" => Dict("longname" => "Locations of the cell faces in the x-direction.",   "units" => "m"),
+        "yF" => Dict("longname" => "Locations of the cell faces in the y-direction.",   "units" => "m"),
+        "zF" => Dict("longname" => "Locations of the cell faces in the z-direction.",   "units" => "m")
     )
 
     # Applies slices to the dimensions d
     for (d, slice) in slice_kw
-        if String(d) in keys(dimensions)
-            dimensions[String(d)] = dimensions[String(d)][get_slice(slice)]
+        if String(d) in keys(dims)
+            dims[String(d)] = dims[String(d)][get_slice(slice)]
         end
     end
 
-    # Writes the sliced dimensions to the specified netcdf file
     Dataset(filename, mode, attrib=attributes) do ds
-        for (dimname, dimarray) in dimensions
-            defVar(ds, dimname, dimarray, (dimname,),
-                   compression=compression, attrib=dim_attrib[dimname])
+        for (dim_name, dim_array) in dims
+            defVar(ds, dim_name, dim_array, (dim_name,),
+                   compression=compression, attrib=dim_attribs[dim_name])
         end
     end
 
     return nothing
 end
 
+const default_output_attributes =
```

Comment:
might be good to add one for `b` (buoyancy)

Reply:
with units `m / s^2`

Reply:
Also we should clarify that `T` is Conservative Temperature and `S` is absolute salinity. This matters for non-linear equations of state.